collect_app/src/main/java/org/odk/collect/android/formlists/savedformlist/DeleteSavedFormFragment.kt

```diff
@@ -46,6 +48,8 @@ class DeleteSavedFormFragment(
             }
         )
     }
+    private var appBarLayout: AppBarLayout? = null
```

Comment:
Same thing here with the fields as for `DeleteBlankFormFragment`.

Reply:
https://github.com/getodk/collect/pull/6390#discussion_r1745303492